### PR TITLE
Card Population into Modal Screen and Article Synopsis Data Pass

### PR DESCRIPTION
**Pull request by htn5**
Issue: [https://trello.com/c/xFg9CrmH]( https://trello.com/c/xFg9CrmH)

When the hotspot button is clicked, the modal page is populated with articles (currently hardcoded to call news API for the articles).

Screen Shot 2022-05-15 at 4 25 30 PM<br>

Each cell is programmed so that when clicked, the news article is displayed on the next page. This data passing is just a placeholder so that we can use data from the modal screen to generate content for the 3 tab view.

Screen Shot 2022-05-15 at 4 25 30 PM<br>

### CODE REVIEW

**Review by K3nnneth — APPROVED** [review]
LGTM

File: GeoNewsFinder4/components/BottomSheet.js [hunk 1]
```diff
@@ -1,25 +1,77 @@
...
     <View style={styles.bottomSheetContainer}>
-      <Text style={styles.modalText}>Heather and Jai need to put articles in here</Text>
-      <TouchableOpacity onPress={navigateToArticleSynopsisScreen} style={styles.openPageButton}>
-        <Text style={styles.openPageButtonText}>Open 3 tab view page</Text>
-      </TouchableOpacity>
       <TouchableOpacity onPress={closeModal} style={styles.closeButton}>
         <Text style={styles.closeButtonText}>Close Bottom Sheet</Text>
       </TouchableOpacity>
+
+      <Text style={styles.modalText}>Heather and Jai need to put articles in here</Text>
```
Comment (mattn02) [inline GeoNewsFinder4/components/BottomSheet.js]: Can we remove this placeholder?

File: GeoNewsFinder4/screens/ArticleSynopsisViewScreen.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import { View, StyleSheet} from 'react-native';
+import { View, Text, TouchableOpacity, StyleSheet, Image } from 'react-native';
```
Comment (mattn02) [inline GeoNewsFinder4/screens/ArticleSynopsisViewScreen.js]: `TouchableOpacity` and `Image` are unused. we should remove it for now to reduce clutter

**Review by mattn02 — CHANGES_REQUESTED** [review]
Everything looks good and well implemented! Need some minor adjustments on code to reduce clutter. Also there is a warning that should be addressed

IMG_07699A3C42BD-1

**Review by mattn02 — APPROVED** [review]
LGTM

**Pull request merged**